backend/de.metas.business/src/test/java/de/metas/costing/methods/AveragePOCostingMethodHandlerTest.java

```diff
@@ -271,6 +271,51 @@ public void inventoryWithQtyAndPrice()
 		assertThat(currentCost.getCostPrice().toBigDecimal()).isEqualTo("10");
 	}
 
+	@Test
+	public void inventoryWithNoQtyAndPrice_NoExplicitCostPrice()
```

Comment:
hmm, not sure about this....
loud thinking: if there are no explicit costs specified, the current cost shall be used...

Reply:
We have current costs if there was already an init with stock. 
That behaviour is tested by https://github.com/metasfresh/metasfresh/blob/e11c4294efdfc81b061fd1b06e830b90c4461d83/backend/de.metas.business/src/test/java/de/metas/costing/methods/AveragePOCostingMethodHandlerTest.java#L364